HLTrigger/Phase2/python/HLT_75e33/psets/initialStepTrajectoryBuilder_cfi.py

```diff
@@ -17,12 +17,16 @@
     lostHitPenalty = cms.double(30.0),
     maxCand = cms.int32(2),
     maxDPhiForLooperReconstruction = cms.double(2.0),
+    maxLostHitsFraction = cms.double(999.0),
+    maxNumberOfHits = cms.int32(100),
     maxPtForLooperReconstruction = cms.double(0.7),
     minNrOfHitsForRebuild = cms.int32(1),
+    minPt = cms.double(0.9),
     propagatorAlong = cms.string('PropagatorWithMaterialParabolicMf'),
     propagatorOpposite = cms.string('PropagatorWithMaterialParabolicMfOpposite'),
     requireSeedHitsInRebuild = cms.bool(True),
     seedAs5DHit = cms.bool(False),
+    seedPairPenalty = cms.int32(0),
```

Comment:
the new parameters are not used by the `GroupedCkfTrajectoryBuilder`, so I would propose not to include them